pkg/config/config.go

```diff
@@ -37,6 +37,7 @@ type Nuke struct {
 type FeatureFlags struct {
 	DisableDeletionProtection  DisableDeletionProtection `yaml:"disable-deletion-protection"`
 	ForceDeleteLightsailAddOns bool                      `yaml:"force-delete-lightsail-addons"`
+	NukeOnFilterFailure        bool                      `yaml:"nuke-on-filter-failure"`
```

Comment:
I couldn't decide which was better:
- Nuking erroneous resources that caused a filter failure (the main feature of this program is to nuke resources, if the program couldn't quite tell, it should default to nuke)
- Filtering erroneous resources (if a user gave a filter they probably don't want a resource nuked, just because we couldn't tell if the resource should be nuked doesn't mean we should destroy it)  

So I just implemented a feature flag